src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -70,8 +75,14 @@ const Sidebar = () => {
           }`}
           onClick={selectPrivateChatTab}
         >
-          {hasUnreadMessages && <div className="notification"></div>}
+          {numberOfUnreadMessages > 0 && <div className="notification"></div>}
```

Comment:
This notification doesn't seem to be showing the count of the unread messages. Should it be?

Reply:
This is just a grey circle... it was there before

Reply:
How does the notification count show in the `NavBar` currently? I would think we might want the same sort of thing maybe? (I envisage it as a 'badge' with the count of unread messages on it, but I don't remember specifically)

Reply:
This part is what I was thinking of/referring to:

- https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/NavBar/NavBar.tsx#L264-L269

Reply:
it looks like this grey circle before the message
![image](https://user-images.githubusercontent.com/40624291/108462810-45ba3900-72a3-11eb-8739-cbfd66ece6d8.png)

Reply:
In the navbar Its looks like this.
![image](https://user-images.githubusercontent.com/40624291/108463890-350ac280-72a5-11eb-8d1e-7e6c50473971.png)

Reply:
![image](https://user-images.githubusercontent.com/40624291/108464185-b2cece00-72a5-11eb-921f-152085b7a4cd.png)

Reply:
@sparkletown/design @sparkletown/product @sparkletown/engineering Do we have designs for this part? How do we want it to look?

Reply:
@jigar-jstechno Currently it looks like the number of unread messages is cut off here as well. That might be something to fix.

![image](https://user-images.githubusercontent.com/753891/108664935-efa9f780-7527-11eb-880a-60f5dadf87de.png)

We'll have to wait on input from @sparkletown/design RE: https://github.com/sparkletown/sparkle/pull/1180/files#r579985293 too

Reply:
We should remove the white dot before "Messages" since we have the new messages count now. Live Schedule is supposed to be removed in the new chat as well so the "unread messages" tab won't be cut off

Reply:
> In the navbar Its looks like this.
> ![image](https://user-images.githubusercontent.com/40624291/108463890-350ac280-72a5-11eb-8d1e-7e6c50473971.png)

I think this topbar link is gone in the new update right ?

Reply:
@mcflyok Yup, the `NavBar` chat icon is gone in the new update.

From our zoom call: No need to have a special badge icon/etc in the sidebar, the white dot can go away. We just want the count of unread messages in the parenthesis.